kitsune/sumo/static/sumo/js/search.js

```diff
@@ -1,9 +1,39 @@
 import "jquery-ui/ui/widgets/tabs";
+import trackEvent from "sumo/js/analytics";
+
 
 $(function() {
   // initiate tabs
-  var tabs = $('#search-tabs').tabs(),
-    cache_search_date = $('.showhide-input');
+  var tabs = $('#search-tabs').tabs();
```

Comment:
I would suggest that all the changes that we need to make should avoid using jquery and use vanilla JS

Reply:
👍  I'll do that. It'll make it more challenging, but it'll keep us moving in that direction.